### ISSUE TITLE
Do we need to load softmax_train.py trained model and then load the pretrained model in pcl_training.py

### ISSUE THREAD

**Issue by machanic** [aamir-mustafa/pcl-adversarial-defense#13]
Does it necessary to firstly softmax_train a classifier and then load this model to train `pcl_training.py`?

**Comment by aamir-mustafa** [comment]
Thanks for the interest in our work.

In our work we trained the network first using the softmax loss and then PCL. In order to replicate the results of our paper, I would recommend you to warm up the network by training it using softmax loss.

Hope that answers the question.
